cmd/producer/main.go

```diff
@@ -16,9 +16,16 @@ func main() {
 	var brokers []string
 	brokers = append(brokers, "localhost:9092")
 
+	var maxBytes = int(2000000)
+	var kafkaVersion = "1.0.2"
+
```

Comment:
I don't have any back ground on these values but IMO it feels like these should be defined in the config package.